wordpress-theme/skyyrose-flagship/assets/css/luxury-theme.css

```
@@ -405,6 +1352,478 @@ label {
+	filter: drop-shadow(0 2px 12px var(--logo-glow-color, rgba(0,0,0,0.15)));
+	transition: transform 0.6s cubic-bezier(0.22,1,0.36,1), filter 0.6s ease;
+}
+
+.cl-logo-svg:hover {
+	transform: scale(1.03);
+}
+
+/* Collection-specific SVG glow */
+.collection-black-rose .cl-logo-svg {
+	--logo-glow-color: rgba(192,192,192,0.2);
+	max-width: 380px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)